docs/tutorials/timeseries/forecasting-indepth.ipynb

```diff
@@ -354,7 +354,7 @@
     "- `SimpleFeedForward`\n",
     "- `TemporalFusionTransformer`\n",
     "\n",
-    "AutoGluon also offers a tree-based global model `AutoGluonTabular`.\n",
+    "AutoGluon also offers a tree-based global model `DirectTabular`.\n",
```

Comment:
I think that "direct" is a somewhat established term for such models, e.g. https://forecastegy.com/posts/multiple-time-series-forecasting-with-scikit-learn/#recursive-vs-direct-forecasting, or [this survey paper](https://arxiv.org/pdf/2202.01712.pdf)

<img width="798" alt="image" src="https://github.com/autogluon/autogluon/assets/6944857/e459d791-fe97-412e-81a9-eb583fb24c7d">

We are not 100% following the above methodology - we train a single model that forecasts all timesteps instead of single model per timestep (otherwise would be computationally infeasible for long horizons).

----
Do you think it would make sense to use "NonRecursive" in this case to differentiate from the above strategies, or are you fine with "Direct" as we are close enough to what people mean by this term?